google/cloud/datacatalog_v1/proto/datacatalog.proto

```diff
@@ -546,15 +522,15 @@ message SearchCatalogRequest {
   // matching to work correctly. See [Data Catalog Search
   // Syntax](https://cloud.google.com/data-catalog/docs/how-to/search-reference)
   // for more information.
-  string query = 1 [(google.api.field_behavior) = REQUIRED];
+  string query = 1 [(google.api.field_behavior) = OPTIONAL];
```

Comment:
Is this change going to cause breakages?

Reply:
Relaxing required->optional is considered safe, though it isn't called out explicitly in the https://google.aip.dev/180 backwards compatibility doc explicitly.